deals/types.go

```diff
@@ -3,14 +3,19 @@ package deals
 import (
 	"os"
 
-	"github.com/filecoin-project/lotus/chain/types"
 	"github.com/ipfs/go-cid"
 )
 
 // StorageDealConfig contains information about a storage proposal for a miner
 type StorageDealConfig struct {
 	Miner      string
-	EpochPrice types.BigInt
+	EpochPrice uint64
```

Comment:
Prefer using simpler types for API interfaces. This change scattered small changes in clients of the API, nothing fancy.